roles/openshift_cluster_monitoring_operator/tasks/install.yaml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Create temp directory for doing work in on target
+  command: mktemp -td openshift-cluster-monitoring-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
+
+- name: Copy files to temp directory
+  copy:
+    src: "{{ item }}"
+    dest: "{{ mktemp.stdout }}/{{ item }}"
+  with_items:
+  - cluster-monitoring-operator.yaml
+  - cluster-monitoring-config.yaml
+
+- name: Copy admin client config
+  command: >
+    cp {{ openshift.common.config_base }}/master/admin.kubeconfig {{ mktemp.stdout }}/admin.kubeconfig
+  changed_when: false
+
+- name: Add monitoring project
+  oc_project:
+    state: present
+    name: openshift-monitoring
+    description: Openshift Monitoring
+
+- name: Add cluster admin permission
+  command: >
+    {{ openshift_client_binary }} adm policy add-cluster-role-to-user cluster-admin -z default
+    --config={{ mktemp.stdout }}/admin.kubeconfig
+    -n openshift-monitoring
+
+- name: Apply the cluster monitoring operator manifests
+  command: >
```

Comment:
consider replacing this `oc apply -f` command with `oc_configmap` and `oc_obj` modules.
See examples:
`oc_configmap`
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/tasks/install_prometheus.yaml#L203-L211

`oc_obj with kind: Deployment`
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/tasks/install_prometheus.yaml#L247-L255

Reply:
Oh, I missed @ironcladlou request to switch to openshift template. In that case, you can ignore this comment.